EnglishFilter/sections/adservers.txt

```diff
@@ -163,6 +163,7 @@
 ||graniteacquitcharacteristic.com^
 ||bestcontentoperation.top^$third-party
 ||balebunsob.com^
+||zergnet.com^$third-party
```

Comment:
in this case better limit the rule by content-type - `script`.
Also the necessary rules are loaded from EasyList. Check that filter.

Reply:
EasyList has these included:
```
||zergnet.com/zerg-inf.js$third-party
||zergnet.com/zerg.js$third-party
```
i suggest complete block of zergnet with `||zergnet.com^$script,third-party`